src/components/Home/MainBlob.tsx

```diff
@@ -0,0 +1,35 @@
+import Image, { StaticImageData } from "next/image";
+import React from "react";
+
+interface BlobProps {
+    image: StaticImageData,
+    headline: string,
+    text: string
+}
+
+
+
+export default function MainBlob (props:BlobProps) {
+ 
+    return (
+        <div className="mainblob flex justify-center items-center">
+        <div className="md:basis-full md:w-3/6 sm:w-5/6 xl:w-2/6">
+            <span className="text-5xl font-bold md:text-3xl xl:text-6xl">{props.headline}</span>
```

Comment:
If you decide to change to my suggested syntax, don't forget to change this and lines 18, 23, 24